uportal-war/src/main/java/org/jasig/portal/rest/MarketplaceRESTController.java

```diff
@@ -96,7 +99,35 @@ public ModelAndView marketplaceEntryFeed(HttpServletRequest request, HttpServlet
         response.setStatus(HttpServletResponse.SC_NO_CONTENT);
         return null;
     }
-
+  
+   
+    @RequestMapping(value="/marketplace/{fname}/ratings", method = RequestMethod.GET)
```

Comment:
Can we adjust this URL slightly to follow the pattern we've been promoting?

Here's an example from DLM:  `/v4-3/dlm/portletRegistry.json`

What about this?  `/v4-3/marketplace/{fname}/ratings.json`